api/v1alpha1/storageconsumer_types.go

```diff
@@ -90,6 +90,10 @@ type ClientStatus struct {
 	// Name is the name of connected storageclient
 	// +optional
 	Name string `json:"name,omitempty"`
+
+	// StorageQuotaUtilizationRatio is the ratio of utilized quota of connected client
+	// +optional
+	StorageQuotaUtilizationRatio string `json:"storageQuotaUtilizationRatio,omitempty"`
```

Comment:
Why a string? we want a ratio which means a floating point number

Reply:
The reason for choosing the string for this is i saw the platform version is string even when its floating so i made it string. Now i checked its values and saw it has multiple decimal points in it(like `4.16.0 / 4.17.0`). I will change to `float32`